include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.cpp

```diff
@@ -189,6 +189,15 @@ TimeIntBDF<dim, Number>::advance_one_timestep_partitioned_solve(bool const use_e
                 dealii::ExcMessage("TemporalDiscretization::BDFCoupledSolution cannot "
                                    "recover velocity and pressure independently."));
   }
+  else if(this->param.temporal_discretization == TemporalDiscretization::BDFConsistentSplittingScheme)
+  {
+    AssertThrow(not(this->param.temporal_discretization ==
+                      TemporalDiscretization::BDFConsistentSplittingScheme and
+                    this->store_solution),
```

Comment:
`store_solution` is not implemented for the `BDFConsistentSplitting` as we only need it for FSI,which depends on ALE, which is not implemented.